io.github.eclab.edisyn.yml

```diff
@@ -0,0 +1,45 @@
+
+id: io.github.eclab.edisyn
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.openjdk11
+command: launch.sh
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
```

Comment:
```suggestion
    - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
```

Reply:
Don't change the substitution order, things from /app are supposed to override /usr

Reply:
It's not like I suggested it https://github.com/flathub/flathub/pull/5762#discussion_r1823404527 :-/

Reply:
Sorry. I fixed this independently of #5762 .   Thanks for the hint. I will commit your suggestion.